README.md

```diff
@@ -71,10 +82,10 @@ Exec all aws commands? (y/N) y
 
 ## Accurate cloudformation matching
 
-By default, iamy will use a simple heuristic (does it end with an ID, eg -ABCDEF1234) to determine if a given resource is managed by cloudformation. 
+By default, iamy will use a simple heuristic (does it end with an ID, eg -ABCDEF1234) to determine if a given resource is managed by cloudformation.
 
 This behaviour is good enough for some cases, but if you want slower but more accurate matching pass `--accurate-cfn`
-to enumerate all cloudformation stacks and resources to determine exactly which resources are managed. 
+to enumerate all cloudformation stacks and resources to determine exactly which resources are managed.
```

Comment:
we can probably remove this from the readme.
I'd like to dump that code all together eventually